src/main/scala/io/github/pauljamescleary/petstore/domain/users/UserRepositoryAlgebra.scala

```diff
@@ -1,19 +1,14 @@
 package io.github.pauljamescleary.petstore.domain.users
 
-import scala.language.higherKinds
+import cats.data.OptionT
+import tsec.authentication.BackingStore
 
-trait UserRepositoryAlgebra[F[_]] {
-  def create(user: User): F[User]
+trait UserBackingStore[F[_]] extends BackingStore[F, Long, User]
```

Comment:
does it make sense to have the repo extend the backing store, or have the UserRepository accept the backing store as a constructor param?  "has a" vs. "is a"

Reply:
The backing store is a generic subset of the UserRepositoryAlgebra I think, seems like our algebra is an extension of it. 
```
  trait BackingStore[F[_], I, V] {
    def put(elem: V): F[V]
    def get(id: I): OptionT[F, V]
    def update(v: V): F[V]
    def delete(id: I): F[Unit]
  }
```